### ISSUE TITLE
Cannot search table header <thead>

### ISSUE THREAD

**Issue by steelywing** [docsifyjs/docsify#1255]
#### Steps to reproduce

Create a `.md`, search `testing4` or `testing5` return no result.

```md
## testing1

testing3

| testing4 | testing5 |
| - | - |
| testing6 | testing7 |
```

or go to https://steelywing.github.io/note/#/SoC?id=ir-infrared search `Wave`

- Docsify version: latest

**Comment by anikethsaha** [comment]
I think  this fix is not working properly 

![image](https://user-images.githubusercontent.com/26347874/86116669-73235e00-baeb-11ea-9950-cc273af101fd.png)


can you check

**Comment by sy-records** [comment]
![image](https://user-images.githubusercontent.com/33931153/86120534-1170ed00-bb07-11ea-897b-f4f199b29484.png)
@anikethsaha fixed via #1256
